tests/package.lisp

```diff
@@ -13,7 +13,9 @@
    (#:analysis #:coalton-impl/analysis)
    (#:entry #:coalton-impl/entry))
   (:export
-   #:run-coalton-tests))
+   #:run-coalton-tests
+   #:run-suite
```

Comment:
I think this might possibly be a confusing name, maybe it should be `#:run-file-tests` or `#:run-test-page` or `#:run-test-file` something like that. Suite would make me assume it's larger than a single file.

Reply:
done, changed to run-test-file